Bottom: added3e2a39a61362f3172130e3be258a87ea756
Top:    02020065c5170632fec3eab7ef686eaaba762143
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-23 17:51:54 +0000

Refresh of fix-broken-undo-handling-for-r

---

diff --git a/src/find.c b/src/find.c
index 107b651..1c9e94a 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3926,7 +3926,7 @@ DRCAll (void)
   TheFlag = FOUNDFLAG | DRCFLAG | SELECTEDFLAG;
 
   if (ResetConnections (true, true))
-    IncrementUndoSerialNumner ();
+    IncrementUndoSerialNumber ();
 
   User = false;
